docs(sources): add integrations directory with 31 source pages (ETL-980) (#739)
glassflow/clickhouse-etl: docs/app/sources/kinesis-firehose/page.mdx

---
title: 'Kinesis Firehose'
description: 'Ingest streaming data from Amazon Kinesis Firehose into ClickHouse with GlassFlow'
---
import { Callout } from 'nextra/components'

# Kinesis Firehose

Firehose is a fully managed delivery service that streams data directly into destinations like S3, Redshift, or OpenSearch. It simplifies ingestion pipelines by handling buffering, batching, and scaling automatically.

## Using Kinesis Firehose with GlassFlow

GlassFlow can ingest data from Kinesis Firehose into ClickHouse with the same pipeline capabilities available to all GlassFlow sources — deduplication, joins, filtering, and stateless transformations.

<Callout type="info">

> **Available on request.** Kinesis Firehose connectors are supported natively via the GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.

Reviewer: ```suggestion
**Available on request.** Kinesis Firehose connectors are natively supported in GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.
```
